docs/examples/video_decode_remap/README.md

```diff
@@ -0,0 +1,83 @@
+# GPU-accelerated video processing in Triton using DALI
+
+This example presents, how to approach GPU-accelerated video processing in Triton using DALI.
+
+### First steps
+
+1. If you do not know yet what DALI is, please refer to [DALI's getting started guide](https://docs.nvidia.com/deeplearning/dali/user-guide/docs/examples/getting_started.html#Overview)
+1. If you are not yet familiar with the idea of using DALI within Triton, please refer to the
+[How to use?](https://github.com/triton-inference-server/dali_backend#how-to-use) guide and
+[Identity example](https://github.com/triton-inference-server/dali_backend/tree/main/docs/examples/identity).
+These two guides explain the concept and are good starting points.
+
+## GPU-accelerated video processing
+
+While in most of data processing libraries (image or video, like PIL, OpenCV etc...), data decoding
+is transparent for user, in DALI we do put emphasis in this step. Actually, whenever `cv2.imread()`
```

Comment:
```suggestion
is transparent for user, in DALI we put emphasis on this step. Actually, whenever `cv2.imread()`
```
1. on rather than in?
2. I am not sure if that much emphasis is necessary. :D